src/python/doc/persistence_graphical_tools_user.rst

```diff
@@ -86,9 +85,9 @@ If you want more information on a specific dimension, for instance:
     # Use subplots to display diagram and density side by side
     fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(12, 5))
     gudhi.plot_persistence_diagram(persistence=pers_diag,
-        axes=axes[0])
+        legend=False, axes=axes[0])
     gudhi.plot_persistence_density(persistence=pers_diag,
-        dimension=1, legend=True, axes=axes[1])
+        legend=True,  axes=axes[1])
```

Comment:
`legend=True` is the default.

Reply:
It was to emphasize the legend when it is True or False, but, yes you are right. I did it on [db99927](https://github.com/GUDHI/gudhi-devel/pull/734/commits/db999275a4b926458f780b5e1c0177ac5fe3f605)